curriculum/challenges/english/25-front-end-development/lab-blog-post-card/66eaddd04a9e533fba689001.md

```diff
@@ -0,0 +1,368 @@
+---
+id: 66eaddd04a9e533fba689001
+title: Design a Blog Post Card
+challengeType: 14
+dashedName: lab-blog-post-card
```

Comment:
```suggestion
dashedName:  design-a-blog-post-card
```

Reply:
As mentioned in chat, this will be updated to the mongo id.